pkgs/applications/networking/xpipe/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv, lib, fetchzip, makeDesktopItem }:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  arch = {
+    x86_64-linux = "x86_64";
+    aarch64-linux = "arm64";
+  }.${system} or throwSystem;
+
+  sha256 = {
+    x86_64-linux = "VmJa0DqnFSzq4wJ+HchguDK4tMeKP/XaE2MidXfootM=";
+    aarch64-linux = "LdXfhZK4n8HqN1u/rKIIZhFk7fREXbTuAzyvvhFR6Lo=";
+  }.${system} or throwSystem;
+
+in stdenv.mkDerivation rec {
+  displayname = "XPipe";
```

Comment:
Does this even do anything, apart from being a common attribute for the desktop item?

Reply:
Yes as I plan to also publish the public test build (PTB) release of xpipe here. The .nix file is autogenerated by the build script and the display name will be altered between normal release and PTB release.

Reply:
No, I mean as an attribute of `mkDerivation`. The option seems to imply that derivations have a  displayname separate from their package name, but AFAIK no such thing exists?

Reply:
Oh then nope

Reply:
Maybe it could be placed in the prepending let block to avoid confusion (or just replaced with literals if it's not going to change anyway)?